src/QGONexum.py

```diff
@@ -112,7 +114,7 @@ def getCoordinates(self) -> tuple[float, float]:
     def changeName(self, new_name: str):
         self.name = new_name
         
-    def __str__(self):
+    def __repr__(self):
```

Comment:
This makes the representation of the nodes, when dumped into the geojson output files, consistent as a strings, instead of a memory location, which changes every time it's run.